### PR TITLE
10.4.0 changelog

### PR DESCRIPTION
**Pull request by chrisbreiding**

### CODE REVIEW

File: content/_changelogs/10.4.0.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Fixed an issue that prevented displayed "Last Updated" information on Windows for git repositories without a commit history. Addressed in [#22741](https://github.com/cypress-io/cypress/issues/22741).
+- Fixed styling of the Latest Runs tooltip to keep it centered over the latest run entry. Fixes [#22629](https://github.com/cypress-io/cypress/issues/22629) and [#22598](https://github.com/cypress-io/cypress/issues/22598).
+- Fixed styling issues with the Specs Explorer at narrow viewport widths. Fixes [#22626](https://github.com/cypress-io/cypress/issues/22626).
+- Fixed a visual issue in the reporter where special characters would be incorrectly formatted with markdown. Fixes [#5679](https://github.com/cypress-io/cypress/issues/5679).
+- Fixed an issue where the Proxy-Authorization header included capitalization that would fail with certain proxies. Fixes [#22383](https://github.com/cypress-io/cypress/issues/22383).
+- Added `react-dom` as a required dependency when setting up a React project for Component Testing. Fixes [#22371](https://github.com/cypress-io/cypress/issues/22371).
```
Comment (emilyrohrbough) [inline content/_changelogs/10.4.0.md]: Isn't this a breaking change to add a required dependency?

File: content/_changelogs/10.4.0.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- The Spec Runner will no longer overflow unexpectedly in Firefox in certain situations. Fixes [#21881](https://github.com/cypress-io/cypress/issues/21881) and [#22610](https://github.com/cypress-io/cypress/issues/22610).
+- Re-added support for the `CYPRESS_NO_COMMAND_LOG` environment variable. Fixes [#21721](https://github.com/cypress-io/cypress/issues/21721) and [#21271](https://github.com/cypress-io/cypress/issues/21271).
```
Comment (emilyrohrbough) [inline content/_changelogs/10.4.0.md]:
```suggestion
- Fixed regression where the `CYPRESS_NO_COMMAND_LOG` environment variable was not honored. Fixes [#21721](https://github.com/cypress-io/cypress/issues/21721) and [#21271](https://github.com/cypress-io/cypress/issues/21271).
```

File: content/_changelogs/10.4.0.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- Using `cy.contains()` with a regular expression now matches submit-type input labels correctly. Fixes [#21166](https://github.com/cypress-io/cypress/issues/21166).
+
+**Miscellaneous:**
+
+- Improved keyboard navigation for Latest Runs data in the Specs Explorer. Addresses [#22628](https://github.com/cypress-io/cypress/issues/22628).
+- Improved accessibility and styling around Specs Explorer tooltips. Addresses [#22597](https://github.com/cypress-io/cypress/issues/22597).
+- Improved the styling and behavior of the Copy button in the Launchpad. Addresses [#21860](https://github.com/cypress-io/cypress/issues/21860) and [#21862](https://github.com/cypress-io/cypress/issues/21862).
```
Comment (emilyrohrbough) [inline content/_changelogs/10.4.0.md]:
```suggestion
- Improved the styling and behavior of the Copy button seen in the Migration workflows. Addresses [#21860](https://github.com/cypress-io/cypress/issues/21860) and [#21862](https://github.com/cypress-io/cypress/issues/21862).
```

**Review by emilyrohrbough — CHANGES_REQUESTED** [review]

File: content/_changelogs/10.4.0.md (hunk 1 above)
Comment (chrisbreiding) [reply content/_changelogs/10.4.0.md]: This only affects the component testing setup wizard. Before it was lacking this dependency, which is required by React, not us.

File: content/_changelogs/10.4.0.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+## 10.4.0
+
+_Released 7/18/2022_
+
+**Features:**
+
+- The Cypress App's sidebar will now automatically collapse at narrow widths. Addressed in [#22393](https://github.com/cypress-io/cypress/issues/22393).
```
[the remark this answers is not in the document]
Comment (chrisbreiding) [reply content/_changelogs/10.4.0.md]: 🤷‍♂️ It was merged as a feature, so it's what bumped this release to 10.4.0.

File: content/_changelogs/10.4.0.md (hunk 1 above)
Comment (emilyrohrbough) [reply content/_changelogs/10.4.0.md]: Gottcya, can we expand on this entry to provide more context?

**Comment by chrisbreiding** [comment]
Going with a patch release instead: https://github.com/cypress-io/cypress-documentation/pull/4612

**Pull request closed without merging**